pkg/diagnostics/client_test.go

```diff
@@ -0,0 +1,78 @@
+package diagnostics
+
+import (
+	"github.com/dcos/dcos-core-cli/pkg/pluginutil"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestClient_CreateErrorsWhenGot503WithStatus(t *testing.T) {
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, "/system/health/v1/report/diagnostics/create", r.URL.String())
+		assert.Equal(t, "POST", r.Method)
+		w.WriteHeader(http.StatusServiceUnavailable)
+		_, err := w.Write([]byte(`{
+			"response_http_code":503,
```

Comment:
nit: Looking at the description of https://github.com/dcos/dcos-diagnostics/pull/162 I think that's a 500 now, isn't it?

Reply:
It's just for test with different values. Before we have some logic about this codes which was wrong (we were assuming 503 means user provided not existing node),